uber/models.py

```diff
@@ -1094,9 +1094,11 @@ class Attendee(MagModel, TakesPaymentMixin):
     dept_checklist_items = relationship('DeptChecklistItem', backref='attendee')
 
     _repr_attr_names = ['full_name']
-    __table_args__ = (
-        UniqueConstraint('badge_num', deferrable=True, initially='DEFERRED'),
-    )
+
+    if Session.engine.dialect == 'postgresql':
```

Comment:
Oh nooooooo. It was too good to be true!

Duly noted -- thanks for catching this. I'll update it (hopefully) by tomorrow.